### DISCUSSION TITLE
Spinner in input.

### DISCUSSION THREAD

**Discussion by gmwill934** [chakra-ui/chakra-ui#2186] [Q&A] [answered]
Hey there. 

I have been scouring the internet to try to implement a spinner inside an Input. Pretty much the input will be visible as the user is typing and making requests to a GraphQL API.

Has anyone has achieved this? I have yet to find examples on the documentation.

**Comment by ljosberinn** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/29307652/95274816-1f8d8280-0847-11eb-81a7-236acf2b0786.png)

Scribbled in the docs real quick.
